docs/assets/basic-install.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: fabricgateways.zalando.org
+  name: "{{{ CRD_PLURAL }}}.zalando.org"
 spec:
   group: zalando.org
   version: v1
   names:
-    kind: FabricGateway
-    singular: fabricgateway
-    plural: fabricgateways
+    kind: "{{{ CRD_KIND }}}"
```

Comment:
This file was meant to have the kind etc hard-coded so people can do `kctl apply -f basic-install.yaml`